Makefile

```diff
@@ -17,11 +19,49 @@ build-base: python-build-script
 	docker build --tag openwisp/openwisp-base:latest \
 	             --file ./build/openwisp_base/Dockerfile ./build/
 
-python-build-script: build.py
-	python build.py change-secret-key
+compose-build: python-build build-base
+	docker-compose build --parallel
```

Comment:
is this one tab or multiple tabs? One tab should be enough, it looks indented a lot

Reply:
It's one tab only.